classifiers/reference_quality/tiktoken_length_classifier/__init__.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import BaseModel
+import tiktoken
+
+INPUT_EXAMPLE = {
+    "text": "The sun is shinng brigt today."
+}
+
+
+class TiktokenLengthClassifierModel(BaseModel):
+    text: str
+
+    class Config:
+        schema_extra = {"example": INPUT_EXAMPLE}
+
+
+def tiktoken_length_classifier(req: TiktokenLengthClassifierModel):
+    """Uses the Tiktoken library to count tokens in a string"""
+    encoding = tiktoken.get_encoding("cl100k_base")
```

Comment:
we could also specify the encoding in the request, don't know how the standard is currently in bricks

Reply:
Yes that would make sense to be able to try out different encodings.